curriculum/js1/5.js

```diff
@@ -5,8 +5,10 @@
  * @param {function} b
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (num, funct, i=0) => {
```

Comment:
i is unnecessary here. How about decrementing num until it hits 0?

Reply:
fixed